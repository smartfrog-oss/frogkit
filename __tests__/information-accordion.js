import { mount } from 'vue-test-utils'
import InformationAccordion from '@/components/information-accordion'

describe('Information Accordion component', () => {

  it('should render component and match snapshot', () => {
    const wrapper = mount(InformationAccordion, {propsData: {title: 'some title goes here'}})
    expect(wrapper.html()).toMatchSnapshot()
  })

})
