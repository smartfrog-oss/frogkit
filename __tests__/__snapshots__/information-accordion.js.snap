// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Information Accordion component should render component and match snapshot 1`] = `
<informationblock placeholder="some title goes here" title="some title goes here" class="fk-information-accordion">
  <div class="fk-information-accordion__content"></div>
</informationblock>
`;
